Description: Add sys.path to Cython includes
 In SageMath this is achieved by patching Cython,
 in this way we can used an unpatched Cython.
Origin: https://git.archlinux.org/svntogit/community.git/tree/trunk?h=packages/sagemath

--- a/sage/src/setup.py
+++ b/sage/src/setup.py
@@ -422,6 +422,7 @@
             build_dir=self.build_dir,
             force=self.force,
             aliases=aliases,
+            include_path=sys.path,
             compiler_directives=self.cython_directives,
             compile_time_env={'PY_VERSION_HEX':sys.hexversion},
             create_extension=sage_create_extension,
--- a/sage/src/sage/misc/cython.py
+++ b/sage/src/sage/misc/cython.py
@@ -277,6 +277,7 @@
 
     # Add cysignals directory to includes
     for path in sys.path:
+        inc.append(path)
         cysignals_path = os.path.join(path, "cysignals")
         if os.path.isdir(cysignals_path):
             inc.append(cysignals_path)
